src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Palindrome {
+
+    /**
+     * Реализуйте метод, проверяющий, является ли заданная строка палиндромом. Палиндромом называется
```

Comment:
@Sofasmile please add <p> and </p> tags on each text section

Reply:
p tags

Reply:
done